examples/azureopenai-private-endpoints/main.tf

```diff
@@ -23,6 +23,38 @@ module "openai" {
       scale_type    = "Standard"
     }
   }
+  diagnostic_settings = {
+     "aoai-svc-diag" = {
+      name = "aoai-svc-diag"
+      log_analytics_workspace_id = "/subscriptions/xxxxxxxxxxxxxxxx/resourceGroups/monitoring-rg/providers/Microsoft.OperationalInsights/workspaces/workspace-xxxxxxxxxxxxxxxxxxxxxxxxxx"
```

Comment:
This fake `log_analytics_workspace_id` with `xxxxxx` cannot be in the example, because the example must be real running code for the end to end testing.

Create in the example the log analytics workspace and pass the real ID value.

Reply:
sorry didn't realize that it test run on real infra